### PR TITLE
scripts/generate/genesis.sh:  some more order in the business

### PR DESCRIPTION
**Pull request by deepfire**
This allows better automation in `iohk-ops`.  Been tested in the last week's three deployments.

### CODE REVIEW

File: scripts/generate/genesis.sh [hunk 1]
```diff
@@ -20,26 +43,54 @@ mkdir $outputDir
...
-if [[ "$N" == "" ]]; then 
+if [[ "$N" == "" ]]; then
     echo "You didn't set N (poor keys amount)";
     exit 1
 fi
 
-F=100 # fake avvm keys
+case "${BUILD_MODE}" in
+nix )
+        IOHKOPS_NIX=${IOHKOPS_DIR}/default.nix
```
Comment (domenkozar) [inline scripts/generate/genesis.sh]: This should now use expressions from the cardano-sl/default.nix

**Comment by domenkozar** [comment]
This also depends on DEVOPS-255 (for example there is no more qanet)

**Comment by georgeee** [comment]
Plus regenerated genesises on top of current master

**Comment by domenkozar** [comment]
https://hydra.iohk.io/eval/62859

**Pull request merged**